pip/flatpak-pip-generator

```diff
@@ -86,6 +118,34 @@ def parse_continuation_lines(fin):
         yield line
 
 
+def fprint(string: str) -> None:
+    separator = '=' * 72  # Same as `flatpak-builder`
+    print(separator)
+    print(string)
+    print(separator)
+
+
+if not opts.python2:
+    minor_version = str(sys.version_info[1])
+    command = [
+        'flatpak',
+        'run',
+        '--command=python3',
+        'org.freedesktop.Platform',
```

Comment:
We don't know this is the runtime they use.

Reply:
I was thinking in doing something like: if there is a  `--runtime` flag use that, else try to use in this order `org.freedesktop.{Platform, Sdk}` and maybe add `org.{gnome,kde}.Platform`. Since this script is no longer compiling packages there is no need to use `Sdk` as first option.

I don't know if this feature should be left entirely optional and doing it baremetal if no runtime is found. It is a reasonable thing to assume the user will have some Platform runtime installed.

Reply:
Fixed in a future pr.